white-test-app-backend/src/main/java/pl/lodz/p/white/whitetestapp/testmanager/manager/TestTemplateContentManager.java

```diff
@@ -16,6 +19,23 @@ public TestTemplateContent getOne(Long id) {
         return repository.getOne(id);
     }
 
+    @Override
+    public TestTemplateContent findOne(Long id) {
+        TestTemplateContent object =  repository.findById(id).orElse(null);
+        return object;
+    }
+
+    @Override
+    public int addQuestionToContent(TestTemplateContent content, Question question) {
+        try {
+            content.getQuestions().add(question);
+            repository.saveAndFlush(content);
+            return 1;
```

Comment:
nie wiem jak to było robione przy innych takich zapisach, ale zamiast 1 mozna by tu zwrócic id nowo utworzonego obiektu
btw jak patrze na tą obsługę błędów poprzez zwracanie 0 itp, to dostaje raka, ale spoko :D